### PR TITLE
Add option to loop forever (nvidia-smi -l)

### PR DESCRIPTION
**Pull request by lanpa**

### CODE REVIEW

**Comment by Stonesjtu** [comment]
Thx for the PR, would you like to review the PR #30 ? I think `-l` can be achieved by aliasing `-i 1`.

**Comment by lanpa** [comment]
I have implement time interval in similar way (`time.sleep()`) before PR, but I found that time for `print_gpustat` depends on different machines, so the user required actual update interval is not satisfied.

#30 looks fine, except that it's behavior is much like `gputop` instead of `gpustat` (lightweight) lol.
How about bringing both of them to the user?

File: gpustat/__main__.py [hunk 1]
```diff
@@ -12,20 +12,26 @@ def print_gpustat(json=False, debug=False, **args):
...
-            import traceback
-            traceback.print_exc(file=sys.stderr)
-        sys.exit(1)
-
-    if json:
-        gpu_stats.print_json(sys.stdout)
-    else:
-        gpu_stats.print_formatted(sys.stdout, **args)
-
+    isloop = args.pop('loop')
```
Comment (Stonesjtu) [inline gpustat/__main__.py]: A `noloop` may be meaningful.

**Comment by Stonesjtu** [comment]
Personally I am not into the `stat` style because it pollute my the output history of my console.

As for the time drifting problem in `-i`, how would you like this solution https://stackoverflow.com/a/25251804

File: gpustat/__main__.py (hunk 1 above)
Comment (lanpa) [reply gpustat/__main__.py]: agree, I was thinking that `store_false` defaults to False...

**Comment by lanpa** [comment]
Since there are already PRs that implement this function in a similar way (#25, #30), closing this.

**Pull request closed without merging**